.github/workflows/post-submit.yaml

```diff
@@ -3,39 +3,36 @@ on:
   push:
     branches:
       - main
-    tags:
-      - 'v*'
 
 permissions:
   contents: write
   pull-requests: read
 
 jobs:
-  push-images:
-    name: Build and push images to quay.io/medik8s
+  push_to_registry:
+    name: Build and push unversioned images to quay.io/medik8s
     runs-on: ubuntu-22.04
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: go.mod
+
```

Comment:
Why is it needed now?

Reply:
I think it was always needed, just not used (and so potentially not using go version set in go.mod). I'll update to version 5

Reply:
>I think it was always needed, 

Can you elaborate on why? afterward, we merely build (and push) the containers. Do we need Golang for that?

Reply:
As part of container build target we also perform tests